### PR TITLE
Adding KoboDbContext to API

### PR DESCRIPTION
**Pull request by rodrigocaballeronexion**
+ Healthcheck
+ Integration Tests

### CODE REVIEW

File: Support.API/Controllers/HealthCheckController.cs [hunk 1]
```diff
@@ -31,6 +35,16 @@ public async Task<IActionResult> HealthCheckDB()
             return BadRequest("DB not accessible");
```
Comment (MarcelKoehlerNexion) [inline Support.API/Controllers/HealthCheckController.cs:35]: quizas cambiar a "Support DB not accessible"

File: Support.API/appsettings.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Server=SUPPORT_DB_INTERNAL_SERVER; Port=SUPPORT_DB_INTERNAL_PORT; Database=SUPPORT_DB_NAME; Username=SUPPORT_DB_USER; Password=SUPPORT_DB_PASSWORD"
+    "DefaultConnection": "Server=SUPPORT_DB_INTERNAL_SERVER; Port=SUPPORT_DB_INTERNAL_PORT; Database=SUPPORT_DB_NAME; Username=SUPPORT_DB_USER; Password=SUPPORT_DB_PASSWORD",
```
Comment (MarcelKoehlerNexion) [inline Support.API/appsettings.json:3]: Quizas mejor "SupportConnection"?

**Review by MarcelKoehlerNexion — APPROVED** [review]
Buenissimo, inclusivo con tests! :D

File: Support.API/Controllers/HealthCheckController.cs (hunk 1 above)
Comment (rodrigocaballeronexion) [reply Support.API/Controllers/HealthCheckController.cs]: Fixed

File: Support.API/appsettings.json (hunk 2 above)
Comment (rodrigocaballeronexion) [reply Support.API/appsettings.json]: Fixed

**Pull request merged**